scripts/bootstrap-desktop.sh

```
@@ -0,0 +1,185 @@
+
+if [[ "${NO_LAUNCH}" -eq 1 ]]; then
+  step "Setup complete (--no-launch)."
+  exit 0
+fi
+
+if [[ "${NO_BRIDGE}" -eq 0 ]]; then
+  step "Starting FastAPI bridge on 127.0.0.1:8765..."
+  (
+    cd "${REPO_ROOT}"
+    # shellcheck disable=SC1091
+    source "${VENV_ACTIVATE}"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)